xrpl/asyncio/transaction/main.py

```diff
@@ -147,13 +147,17 @@ async def autofill_and_sign(
 async def submit(
     transaction: Transaction,
     client: Client,
+    *,
+    fail_hard: bool = False,
 ) -> Response:
     """
     Submits a transaction to the ledger.
 
     Args:
         transaction: the Transaction to be submitted.
         client: the network client with which to submit the transaction.
+        fail_hard: an optional boolean. If True, and the transaction fails locally,
```

Comment:
I'm not quite sure what this means by locally - my understanding is that this failure is due to the initial server rejecting you, and so it's not local. (Maybe better phrasing could be "fails for the initial server"?)

Reply:
Just talked with Elliot, locally means locally to the initial server (once the tx fail, the server will not send it to other servers to retry), so keeping it as is similarly to xrpl.js is fine.

Reply:
I think that if you have to ask what it means we should change the wording - it should be clear to people who don't have that additional context that "if it fails for the initial server it will not send to other servers to retry"